misk/web/tabs/web-actions/src/ducks/webActions.ts

```diff
@@ -9,6 +9,41 @@ import { HTTPMethod } from "http-method-enum"
 import { chain } from "lodash"
 import { all, AllEffect, call, put, takeLatest } from "redux-saga/effects"
 
+export const enum TypescriptBaseTypes {
+  "boolean" = "boolean",
+  "number" = "number",
+  "string" = "string",
+  "enum" = "enum",
+  "any" = "any",
+  "null" = "null"
+}
+
+export const enum KotlinTypes {
```

Comment:
should we be expressing all the types here? https://github.com/square/misk/blob/1e5bb123c2e85118a205d1e040e99cb111531dbf/misk/src/main/kotlin/misk/web/RequestTypes.kt#L73-L81

Reply:
Absolutely 👍 I'll add a commit that adds this.